roles/neutron-data-network/tasks/main.yml

```diff
@@ -59,6 +59,50 @@
     neutron-metadata-agent:
       config_files: /etc/neutron/metadata_agent.ini
 
+- name: neutron lbaas config
+  template:
+    src=etc/neutron/neutron_lbaas.conf
+    dest=/etc/neutron/neutron_lbaas.conf
+    mode=0644
+  notify:
+     - restart neutron services
+  when: neutron.lbaas.enable
+
+- name: neutron lbaas agent config
+  template:
+    src=etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini
+    dest=/etc/neutron/services/loadbalancer/haproxy
+    mode=0644
+  notify:
+     - restart neutron services
+  when: neutron.lbaas.enable
+
+- name: update-alternatives for lbaas
+  shell:
+    update-alternatives --install /usr/local/bin/neutron-lbaasv2-agent \
+    neutron-lbaasv2-agent \
+    /opt/openstack/current/neutron/bin/neutron-lbaasv2-agent 10 --force
+  when: neutron.lbaas.enable and openstack_install_method == 'package'
```

Comment:
Updating alternatives should be done in the same place the rest of it happens for a package install. Just because we put the executable in place doesn't mean the system will be configured to use it, so we shouldn't split up this type of task across multiple files.